src/components/CopyButton/CopyButton.tsx

```diff
@@ -87,36 +84,19 @@ class CopyButton extends React.PureComponent<Props, State> {
     this.props.onClick(event)
   }
 
-  isIconOnly = () => {
```

Comment:
I removed some of this stuff as we only have one version of this button that we use everywhere.